dapps/onboarding.vue/src/components/accept-contact/accept-contact.ts

```diff
@@ -133,13 +133,15 @@ export default class AcceptContact extends mixins(EvanComponent) {
         accountId: this.runtime.activeIdentity,
       });
 
-
       // load my address book
       await this.runtime.profile.loadForAccount(this.accountId,
         this.runtime.profile.treeLabels.addressBook);
       // search for the target public key
-      const targetProfile = await bccHelper.getProfileForAccount(bcc,
```

Comment:
Any reason why this helper function has to be replaced?

Reply:
This was the actual bug of this pr. This function does not exist anymore :D

Reply:
Haha, great :D